tests/pretrained_test.py

```diff
@@ -112,30 +110,28 @@ def test_semantic_role_labeling(self):
             },
             {
                 "verb": "will",
-                "description": "[ARGM-ADV: If you liked the music we were playing last "
-                "night] , [ARG0: you] [V: will] [ARG1: absolutely love what "
-                "we 're playing tomorrow] !",
+                "description": "If you liked the music we were playing last night , you [V: will] absolutely love what we 're playing tomorrow !",
                 "tags": [
-                    "B-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
-                    "I-ARGM-ADV",
                     "O",
-                    "B-ARG0",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
+                    "O",
                     "B-V",
-                    "B-ARG1",
-                    "I-ARG1",
-                    "I-ARG1",
-                    "I-ARG1",
-                    "I-ARG1",
-                    "I-ARG1",
-                    "I-ARG1",
```

Comment:
This is arguably a regression. We can't guarantee that our models won't ever regress on test cases when we re-train them, so I'll merge this anyways.

Reply:
I would classify this as an improvement, actually.  "Will" here is a modal verb, it shouldn't have any arguments on its own.  It's a failure of the heuristic in the predictor that this gets an entry in the result list at all.